CHANGELOG.md

```diff
@@ -222,152 +226,88 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- TfraEntry Time and MoofOffset types changed to unsigned
-- TfraEntr attribute name SampleDelta corrected to SampleNumber
-
-### Added
+- Improved uuid box handling
+- Improved esds box and underlying descriptor handling
+- Extended decryption example with cbcs encryption
+- Improved the decryption example with in-place cenc decryption
 
-- MediaSegment and Fragment have new StartPos attribute
-- mp4.File now has Mfra pointer
-- MfraBox has new method FindEntry
-- MediaSegment, Fragment, and Trun method CommonSampleDuration
-- Added two MSS UUID constants
+## [0.38.0] - 2022-11-17
 
 ### Fixed
 
-- fix AVC slice header parsing #272
-- mp4ff-wvttlister works with Unified Streaming wvtt ismt file
-- Fragment.GetFullSamples() allows tfdt to be absent
-- Fragment.GetFullSamples() defaults to offset being moof
-- mp4ff-wvttlister works for Unified Streaming wvtt asset
-- mp4crop now crops elst entries
-- mp4crop now handles multiple sample durations correctly
-- HEVC SPS parsing details
+- Only start new segment at start or styp box
 
-## [0.38.1] - 2023-09-22
+## [0.37.0] - 2022-09-22
```

Comment:
what happened here?

Reply:
Lots of strange changes when trying out AI tools. Will revert to only have the right delta.

It is important to double-check.

Thanks for pointing it out.